components/04-views/product-view/_product-view.scss

```diff
@@ -322,6 +328,9 @@ $product-view__wrapper-margin-bottom\@medium            : 40px !default;
         @include mq($screen-m) {
             float: none;
```

Comment:
Do we need this property. I know it's legacy code but please check it.

Reply:
Still there is `float: none;`

Reply:
done